### PR TITLE
Remove "lerna" peer dependency

### PR DESCRIPTION
**Pull request by Turbo87**

### CODE REVIEW

File: src/progressBar.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    // Intentionally a noop because node-progress doesn't work well in non-TTY
+    // environments
+    if (!process.stdout.isTTY) {
+      return;
+    }
+
+    // Don't do any of this while testing
+    if (process.env.NODE_ENV === "lerna-test") {
```
Comment (hzoo) [inline src/progressBar.js]: Hmm I guess we don't set this in this repo

File: package.json [hunk 2]
```diff
@@ -48,8 +50,5 @@
     "jest": "^18.1.0",
     "lerna": "^2.0.0-beta.32",
```
Comment (hzoo) [inline package.json]: I guess we can remove this too?

**Comment by Turbo87** [comment]
@hzoo thanks. comments should be addressed.

**Comment by Turbo87** [comment]
forgot to remove the imports from the test files. tests are 🍏  now though!

**Comment by gigabo** [comment]
Looks good to me.  It's kind of a bummer to have the progress-bar code duplicated with Lerna.  Will have to keep an eye out for bug fixes and make sure they get pulled across.

**Comment by hzoo** [comment]
Yeah maybe we can just make a repo for it if it's necessary

**Pull request merged**